### DISCUSSION TITLE
[URGENT] Activate Github for Student

### DISCUSSION THREAD

**Discussion by faysalhossain2007** [community/community#35405] [GitHub Education]
Hi,

I have uploaded the image of my academic enrollment certificate but it is still saying my coupon has expired. I am a PhD student in UVA for 5+ years. My new Student ID doesn't have any valid date information. 

What will be the other option?

**Comment by Alinshan** [comment]
add your admission form which include date and signature of the institution

**Comment by samus-aran** [comment]
Hi @faysalhossain2007 👋 

We would love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by faysalhossain2007** [comment]
I have already added my enrollment information. not sure whether it looks good  to you or not.

**Comment by faysalhossain2007** [comment]
should I upload my enrollment certificate here?

**Comment by faysalhossain2007** [comment]
Any help??

**Comment by faysalhossain2007** [comment]
btw Github hasn't sent me any rejection letter. Actually they sent me another notice that my github coupon has been expired. And this time when I try to upload my enrollment certificate, it says this document is already in their system. I need to upload a new one.

**Comment by backtoaldo** [comment]
I had several problems sending my documentation, three times it was not accepted, I tried it on mobile - unfortunately it didn't work. From the computer, deleting all files, I started a new attempt with the file in jpg - and changing the filename to my name and course, it ended up being accepted. I don't know how it works, with a friend it was on the first try, keep trying, it will probably work.
